# エラーステータス(SCIBOT-X、L-MESのみ)SET
uint8 LastFailD
uint8 LastFailC
uint8 LastFailN
uint8 LastFailE
---

